README.md

```diff
@@ -125,16 +116,10 @@ make -j$(nproc)
 </details>
 <details><summary>macOS</summary>
 
-Install the dependencies using [Homebrew](https://brew.sh/):
-```
-brew install cmake sdl2_mixer sdl2_ttf libsodium pkg-config
-```
-### Compiling
+Make sure you have [Homebrew](https://brew.sh/) installed, then run:
+
 ```
-mkdir build
-cd build
-cmake ..
-make -j$(sysctl -n hw.physicalcpu)
+./macos-build.sh --build-all-x64

```

Comment:
I mostly see xcode-build.sh as a temp solution until we have 64bit sorted out.

Reply:
Cool, I've restored the 64-bit build instructions mostly to the previous state, but also added the `brew bundle` invocation.